src/main/resources/de/tracetronic/jenkins/plugins/ecutest/report/atx/pipeline/ATXNewServerStep/help.html

```diff
@@ -30,33 +30,33 @@
     <dl>Examples:
         <dd>
             <pre>
-ATX.newServer('TEST-GUIDE', 'ECU-TEST')
+ATX.newServer('test.guide', 'ecu.test')
             </pre>
             <pre>
-ATX.newServer('TEST-GUIDE', 'ECU-TEST', 'http://localhost:8085', false, '', '1')
+ATX.newServer('test.guide', 'ecu.test', 'http://localhost:8085', false, '', '1')
             </pre>
             <pre>
-ATX.newServer atxName: 'TEST-GUIDE', toolName: 'ECU-TEST', fullServerURL: 'https://127.0.0.1:8086/context', uploadToServer: true
+ATX.newServer atxName: 'test.guide', toolName: 'ecu.test', fullServerURL: 'https://127.0.0.1:8086/context', uploadToServer: true
```

Comment:
Do we also need the toolName "test.guide"?

Reply:
test.guide is not a tool installation but ecu.test (<jenkins url>/configureTools)